tests/data_flow_test.go

```diff
@@ -43,7 +47,7 @@ func Test_Dataflow(t *testing.T) {
 
 	testServer := newTestServer(listener, func(conn *testServerConn) error {
 		for {
-			name, eof := conn.ReadString(1024, 1*time.Minute)
+			name, eof := conn.ReadString(math.MaxUint16*4, 1*time.Minute)
```

Comment:
out of curiosity - why MaxUint16 * 4 instead of just "1024"? same comment wrt 512 below vs MaxUint16*2... seems... strange. makes me think there's a reason?